milestone2.ts

```diff
@@ -139,23 +142,23 @@ type UpdateVolunteer = (
   vol: Volunteer,
   updates: Partial<Volunteer>
 ) => Volunteer;
-let updateVolunteer: UpdateVolunteer; // Code here
+let updateVolunteer: UpdateVolunteer = (vol, updates) => ({...vol, ...updates}); // Code here
 
-// let kyleNew = updateVolunteer(kyleClone);
-//console.log(kyleNew);
+// kyleNew = updateVolunteer(kyleClone, updatedLocation);
+// console.log(kyleNew);
 
 // Question 7: Object Destructuring
 /* Now that we have our updated kyle, lets use object destructuring to get his name, age, and city
  */
 
 type GetVolunteerInfo = (vol: Volunteer) => String;
 let getVolunteerInfo: GetVolunteerInfo = (vol) => {
-  let name, age, city; // Code here
+  let [name, age, city] = [vol.name, vol.age, vol.city];// Code here
```

Comment:
It looks like you aren't using object destructuring here but instead using array destructuring. Please redo this question with object destructuring instead (with the curly brackets) and resubmit. You can find an example of object destructuring in the demo from session 3 here: https://h4i.notion.site/Intro-to-Javascript-83f29480d7c14b0c9163212b96f6ab0f